quest_maker/templates/base.html

```diff
@@ -43,17 +43,34 @@
             <span class="icon-bar"></span>
             <span class="icon-bar"></span>
           </button>
-          <a class="brand" href="#">Quest Maker</a>
+          <a class="brand" href="{% url 'quest_maker_app:homepage' %}">Quest Maker</a>
           <div class="nav-collapse collapse">
             <ul class="nav">
               <li class="active"><a href="{% url 'quest_maker_app:homepage' %}">Home</a></li>
               <li><a href="{% url 'quest_maker_app:about' %}">About</a></li>
             </ul>
+
+            {% if user.is_authenticated %}
+              <a href="{% url 'quest_maker_app:logout' %}"
+                class="btn btn-info navbar-form pull-right"
+                role="button">Log Out</a>
+            {% else %}
+              <a href="{% url 'quest_maker_app:signup' %}"
+                class="btn btn-info navbar-form pull-right"
+                role="button">Sign Up</a>
+
+              <a href="{% url 'quest_maker_app:login' %}"
+                class="btn btn-info navbar-form pull-right"
+                role="button">Log In</a>
+            {% endif %}
+
+<!--
```

Comment:
That's fine. I thought we might want to add nav-bar login back at some point, but perhaps not.